README.md

```diff
@@ -1 +1,28 @@
-# swift-differentiation-testing
\ No newline at end of file
+# swift-differentiation-testing
+This repository serves as a test suite of crashes and other broken behaviour related to the [Differentiable Swift](https://github.com/differentiable-swift#meet-differentiable-swift) language feature. It aims to provide a low bar for Differentiable Swift users to provide reproducers to crashes or incorrect behaviour. 
+Tests that reproduce currently broken behaviour are added as expecting to fail and once fixed in future toolchain releases will be marked as fixed and used to run as a test suite. 
```

Comment:
```suggestion
This repository serves as a tracker suite of crashes and other broken behaviour related to the [Differentiable Swift](https://github.com/differentiable-swift#meet-differentiable-swift) language feature. It aims to provide a low bar for Differentiable Swift users to provide reproducers to crashes or incorrect behaviour.
Tests that reproduce currently broken behaviour are added as expecting to fail and once fixed in future toolchain releases will be marked as fixed and used to run as a test suite. 
```

Reply:
To me it's more of a test suite.

Reply:
Mostly semantics. Test suite seems fine.